client/src/context/toggle.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { useCookies } from "react-cookie";
+
+const ToggleContext = React.createContext({
+  showGraded: false,
+  onlyCreator: false,
+  toggleGraded: () => {
+    1 + 1;
```

Comment:
are the `1+1`'s to get around some eslint check? If that's the case, better to just use an `eslint-disable-next-line` if that's an option